src/ConnectedVMware/test/AzConnectedVMwareVNet.Tests.ps1

```diff
@@ -0,0 +1,30 @@
+if(($null -eq $TestName) -or ($TestName -contains 'AzConnectedVMwareVNet'))
+{
+  $loadEnvPath = Join-Path $PSScriptRoot 'loadEnv.ps1'
+  if (-Not (Test-Path -Path $loadEnvPath)) {
+      $loadEnvPath = Join-Path $PSScriptRoot '..\loadEnv.ps1'
+  }
+  . ($loadEnvPath)
+  $TestRecordingFile = Join-Path $PSScriptRoot 'AzConnectedVMwareVNet.Recording.json'
+  $currentPath = $PSScriptRoot
+  while(-not $mockingPath) {
+      $mockingPath = Get-ChildItem -Path $currentPath -Recurse -Include 'HttpPipelineMocking.ps1' -File
+      $currentPath = Split-Path -Path $currentPath -Parent
+  }
+  . ($mockingPath | Select-Object -First 1).FullName
+}
+
+Describe 'AzConnectedVMwareVNet' {
+    It 'CreateExpanded' -Skip {
```

Comment:
```suggestion
    It 'CreateExpanded' {
```
The skip tag doesn't be removed. I suspect all test cases aren't recorded successfully. Please remove skip tag for all test cases and re-run test-module.ps1 -Record

Reply:
Fix